### PR TITLE
fix the action conversion

### PR DESCRIPTION
**Pull request by TonybynMp4**
Fix the action parameter to use local (qbx-radialmenu) functions and fix how it checks for the different options.

Might want an option for exports too i guess, but i don't really see how to do that

### CODE REVIEW

**Comment by BerkieBb** [comment]
The purpose of the action parameter is to execute functions sent through the parameter, not execute an existing function inside the radialmenu resource

**Comment by TonybynMp4** [comment]
> The purpose of the action parameter is to execute functions sent through the parameter, not execute an existing function inside the radialmenu resource

oh that makes more sense lol

**Comment by TonybynMp4** [comment]
> The purpose of the action parameter is to execute functions sent through the parameter, not execute an existing function inside the radialmenu resource

Honestly it could do both 🤔 
but for now it fixes it anyway

File: client/main.lua [hunk 1]
```diff
@@ -20,11 +20,10 @@ local function convert(tbl)
             action = function() TriggerEvent(tbl.event, tbl.arg or nil) end
         elseif tbl.type == 'server' then
             action = function() TriggerServerEvent(tbl.event, tbl.arg or nil) end
-        elseif tbl.action then
-            action = function() tbl.action(tbl.arg) end
         end
-    end
-    if tbl.command then
+    elseif tbl.action then
+        action = tbl.action(tbl.arg or nil)
```
Comment (BerkieBb) [inline client/main.lua]: Leaving out the `or nil` does the same

**Review by BerkieBb — APPROVED** [review]

**Review by FjamZoo — APPROVED** [review]

**Pull request merged**